### ISSUE TITLE
Mouse Hover is not ignored by IgnorePointer

### ISSUE THREAD

**Issue by creativecreatorormaybenot** [flutter/flutter#60282]
Currently, using an [`IgnorePointer`](https://api.flutter.dev/flutter/widgets/IgnorePointer-class.html) will **not** ignore mouse _hover_ events.

## Use case

Sometimes, we want to ingore hit events on a sub tree → `IgnorePointer`. In Flutter web, we might also want to ignore mouse hover events and I would say this is even _expected_ behavior for `IgnorePointer`.

## Workaround

This is the workaround I came up with:

```dart
Stack(
  children: [
    subtree,
    const MouseRegion(
      child: SizedBox.expand(),
    ),
  ],
);
```

**Comment by yjbanov** [comment]
This doesn't sound like web-specific behavior, but more of a framework issue.

/cc @dkwingsmt

**Comment by dkwingsmt** [comment]
A duplicate of https://github.com/flutter/flutter/issues/35213. This is blocked by https://github.com/flutter/flutter/pull/59883 which I'm working on.

**Comment by creativecreatorormaybenot** [comment]
@dkwingsmt Awesome! It always amazes me how hard it is to find these specific GitHub issues via search.

I knew this issue had to exist, but I could not find it at all.
